database/database.go

```diff
@@ -0,0 +1,104 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+)
+
+// Db variable use in other package
+var Db *sql.DB
+
+const (
+	dbhost = "DBHOST"
+	dbport = "DBPORT"
+	dbuser = "DBUSER"
+	dbpass = "DBPASS"
+	dbname = "DBNAME"
+)
+
+// InitDb will start DB connection
+func InitDb() {
+	config := dbConfig()
+	var err error
+	psqlInfo := fmt.Sprintf("host=%s port=%s user=%s "+
+		"password=%s dbname=%s sslmode=disable",
+		config[dbhost], config[dbport],
+		config[dbuser], config[dbpass], config[dbname])
+
+	Db, err = sql.Open("postgres", psqlInfo)
+	if err != nil {
+		panic(err)
+	}
+	err = Db.Ping()
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println("Successfully connected to Database!")
```

Comment:
Let us use some logging library e.g. glog
We might not want to use Print functions for logging